solana/rpc/api.py

```diff
@@ -19,6 +20,8 @@ class Client:  # pylint: disable=too-many-public-methods
     """Client class."""
 
     _comm_key = "commitment"
+    _encoding_key = "encoding"
```

Comment:
All you replace all instances of `"encoding"` with this key.

Reply:
Done.